common/src/main/java/com/wynntils/core/consumers/features/FeatureManager.java

```diff
@@ -449,29 +450,37 @@ public void disableFeature(Feature feature) {
 
         feature.onDisable();
 
+        if (overwriteConfig) {
+            feature.setUserEnabled(false);
```

Comment:
this will end up calling disableFeature() recursively. That seems like a bad thing. Maybe @kristofbolyai has any  suggestions? Or maybe we should just drop this, and hope this resolves itself when I rewrite this.

Reply:
I don't think it calls infinitely, just twice at most. What is your plan on the rewrite? Before 1.20.x or after?

Reply:
The rewrite has already started; I'm pushing it incrementally. But I can't say when I will be done. Otoh, we don't have a plan for 1.20 either, right? (Not that I know about anyway...) So the answer is: possibly, maybe likely, or perhaps not :).